src/extension/content-script/batteries/VimeoVideo.ts

```diff
@@ -0,0 +1,48 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+const urlMatcher = /^https:\/\/vimeo.com\/.*/;
+
+const battery = (): void => {
+  document.getElementsByClassName("first")[0].innerHTML;
+  const videoDescription = document.querySelector(".first");
```

Comment:
it seems here `.first` is not enough?
<img width="743" alt="image" src="https://user-images.githubusercontent.com/318/151153584-3d720eb8-ca19-4700-a35e-2d9357193bd1.png">

Reply:
updated. not sure about etiquette, should I resolve the conversation myself or wait until you review?